### PR TITLE
Redirect workflow submit to process detail

### PR DESCRIPTION
**Pull request by tjeerddie**
- Change form submit in process detail to stay on page instead of redirect

### CODE REVIEW

File: src/pages/ProcessDetail.tsx [hunk 1]
```diff
@@ -123,7 +123,7 @@ class ProcessDetail extends React.PureComponent<IProps, IState> {
         let enrichedProcess = processInstance as ProcessWithDetails;
 
         const stepUserInput: InputForm | undefined = enrichedProcess.form;
-        const tabs = stepUserInput ? this.state.tabs : ["process"];
+        const tabs = stepUserInput ? ["user_input", "process"] : ["process"];
```
Comment (Mark90) [inline src/pages/ProcessDetail.tsx:126]: What does this change do?

File: src/pages/NewTask.tsx [hunk 2]
```diff
@@ -60,7 +60,7 @@ class NewTask extends React.Component<WrappedComponentProps, IState> {
         const { select_task } = taskInput[0] as { select_task: string };
 
         let promise = this.context.apiClient.startProcess(select_task, taskInput.slice(1)).then((process) => {
-            this.context.redirect(`/tasks?highlight=${process.id}`);
+            this.context.redirect(`/processes/${process.id}`);
```
Comment (Mark90) [inline src/pages/NewTask.tsx:63]: Not really in scope of this change, but perhaps we should also have a `/tasks/<pid>` route in the app and make tasks more distinct from processes (at least in the frontend, on the backend they're almost identica of course)

**Review by Mark90 — APPROVED** [review]

File: src/pages/ProcessDetail.tsx (hunk 1 above)
Comment (tjeerddie) [reply src/pages/ProcessDetail.tsx:126]: this is so when the `state.tabs` changes `["process"]` it doesnt become ` stepUserInput ? ["process"] : ["process"];`

File: src/pages/NewTask.tsx (hunk 2 above)
Comment (tjeerddie) [reply src/pages/NewTask.tsx:63]: yea, maybe, lets discuss next meeting.

**Pull request merged**